bin/root-transition.sh

```diff
@@ -151,19 +154,67 @@ transition() {
   echo "Please save them safely and privately."
 }
 
+check_version_namespace() {
+  OUT=`kubectl get po -n $1 -o yaml | grep "istio\/proxyv2\:1\."`
+
```

Comment:
It's a good practice use `local` variables in functions.

Reply:
Done.